00 Boilerplate/package.json

```diff
@@ -9,18 +9,18 @@
   },
   "author": "",
   "license": "ISC",
-  "devDependencies": {
-    "typescript": "^2.0.3",
-    "webpack": "^1.13.2",
-    "webpack-devserver": "0.0.6"
-  },
   "dependencies": {
     "bootstrap": "^3.3.7",
     "css-loader": "^0.25.0",
     "file-loader": "^0.9.0",
-    "html-webpack-plugin": "^2.22.0",
+    "html-webpack-plugin": "^2.24.0",
     "style-loader": "^0.13.1",
-    "ts-loader": "^0.9.3",
-    "url-loader": "^0.5.7"
+    "ts-loader": "^0.9.5",
+    "url-loader": "^0.5.7",
+    "webpack": "^1.13.2"
+  },
+  "devDependencies": {
+    "typescript": "^2.0.3",
+    "webpack-devserver": "0.0.6"
   }
```

Comment:
For this example, in dependecies should be only Bootstrap, and others dependencies should be installed like devDependencies:

dependencies --> "bootstrap"

devDependencies --> "css-loader", "file-loader", "html-webpack-plugin", "style-loader", "ts-loader", "url-loader", "webpack", "typescript", "webpack-devserver"